CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Supplying api token manually is now supported
+- Add support for OIDC authentication
+- Add support for enabling and disabling of an authenticator
+- Add support for creating hosts using host factory with provided annotations
+- The `get_server_info` method is now available in SDK. It is only supported against Conjur enterprise server
+
```

Comment:
These should be under the `## [8.0.0]` heading because of the new automated release process. More info [here](https://github.com/conjurinc/docs/blob/master/reference/infrastructure/automated_releases.md#development-conventions). Also please include link to the PR.